lm_eval/evaluator.py

```diff
@@ -41,6 +40,7 @@ def simple_evaluate(
     decontamination_ngrams_path=None,
     write_out: bool = False,
     log_samples: bool = True,
+    verbosity: str = "INFO",
```

Comment:
I think `verbosity` kwarg here doesn't ever get utilized?

Reply:
Also we maybe want to do something like "call `lm_eval.tasks.initialize_tasks()` if it has not already been called", within `evaluate()` and `simple_evaluate()`. Or if not, we need to put into the docs that one must have the line with `initialize_tasks()` called prior to getting a task dict / calling the evaluate methods